frontend/src/pages/Dashboard/Dashboard.module.css

```
@@ -1,11 +1,15 @@
 .dashboardGrid {
+  --top-row-height: 188px;
+  --grid-gap: 15px;
+  --right-top-row: 360px;
+  --right-bottom-row: 120px;
   width: 1273px;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)